src/main/java/mate/controller/driver/AddDriverController.java

```diff
@@ -25,8 +25,15 @@ protected void doPost(HttpServletRequest req, HttpServletResponse resp)
             throws IOException {
         String name = req.getParameter("name");
         String licenseNumber = req.getParameter("license_number");
-        Driver driver = new Driver(name, licenseNumber);
-        driverService.create(driver);
-        resp.sendRedirect("/drivers/add");
+        String login = req.getParameter("login");
+        String password = req.getParameter("password");
+        String passwordAgain = req.getParameter("password_again");
+        if (password.equals(passwordAgain)) {
+            Driver driver = new Driver(name, licenseNumber, login, password);
+            driverService.create(driver);
+            resp.sendRedirect("/drivers/add");
+        } else {
+            resp.sendRedirect("/drivers/");
```

Comment:
you can't go to /drivers if passwords are not equal, you should redirect to /drivers/add again with an error message perfectly